src/local_device.rs

```diff
@@ -267,6 +264,16 @@ impl LocalDevice {
         Ok(SecretKey(self.0.local_symkey.clone()))
     }
 
+    #[getter]
+    fn time_provider(&self) -> PyResult<TimeProvider> {
+        Ok(TimeProvider(self.0.time_provider.clone()))
+    }
+
+    // TODO: rename this into `now`
+    fn timestamp(&self) -> PyResult<DateTime> {
```

Comment:
![Do IT](https://user-images.githubusercontent.com/30697622/186360552-84af7549-0896-49af-9a48-c522cf01aaf2.png)